app/models/manageiq/providers/ibm_power_vc/inventory/parser/cloud_manager.rb

```diff
@@ -3,6 +3,18 @@ def parse_vm(vm, hosts)
     super
     server = persister.vms.find_or_build(vm.id.to_s)
 
+    pg =  placement_group_by_vm_id[vm.id.to_s]
+    placement_group_name = ""
+    placement_group_name = pg.name unless pg.nil?
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => N_('Placement Group'),
+      :name         => 'placement_group',
+      :value        => placement_group_name,
+      :read_only    => true
+    )
+    
+
```

Comment:
I thought we were trying to get away from using advanced_settings for this, should this be
```suggestion
    server.placement_group = persister.placement_groups.lazy_find(pg) if pg
```

Reply:
Also, does this belong in the openstack parser?  Doesn't appear to be any powervc specific logic here

Reply:
my bad. I did not remove it from here.